userspace/falco/outputs_http.cpp

```diff
@@ -34,11 +34,14 @@ void falco::outputs::output_http::output(const message *msg)
 		} else {
 			slist1 = curl_slist_append(slist1, "Content-Type: text/plain");
 		}
+		slist1 = curl_slist_append(slist1, m_oc.options["user_agent"].c_str());
```

Comment:
By testing it locally, I was not able to get the user agent header (but maybe it was me). Either way, not setting the `user-agent: ` prefix properly would result in an invalid header I think. Would it be better to use this instead?
```
curl_easy_setopt(curl, CURLOPT_USERAGENT, m_oc.options["user_agent"].c_str());
```

Reply:
i'll have to retest this in my setup. I was pretty sure that worked on my case, when checking logs on remote end.

Reply:
Oh, i think i missd the "User-Agent:" string from the header, since i used a predefined string before.